specification/ml/post_data/MlPostJobDataResponse.ts

```diff
@@ -23,15 +23,15 @@ import { integer, long } from '@_types/Numeric'
 export class Response {
```

Comment:
Not touched by you, but the timestamps should probably by `EpocTime<UnitMillis>`.